SpotifyAPI/Local/Models/SpotifyUri.cs

```diff
@@ -0,0 +1,23 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace SpotifyAPI.Local.Models
+{
+    public class SpotifyUri
+    {
+        public string Base { get; internal set; }
+        public string Type { get; internal set; }
+        public string Id { get; internal set; }
+
+        public SpotifyUri(string Uri)
+        {
+            string[] props = Uri.Split(':');
```

Comment:
Yea, `ArgumentException ` sounds like the correct way to go.
I'm not aware of them, but still, I think there is just no URI at all sometimes